src/app/modules/notification/components/notifications/notifications.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="notifications" id="notifications" class="notifications">
+  <div *ngFor="let n of notifications"
```

Comment:
I don't like so much this type of naming (`n`), it's clearer to have `notification`.
You for is little so It's not that serious, but generally it's better to have understandable names ;)